st2common/st2common/rbac/loader.py

```diff
@@ -114,6 +117,26 @@ def load_user_role_assignments(self):
 
         return result
 
+    def load_group_to_role_maps(self):
+        """
+        Load all the remote group to local role mappings.
+
+        :rtype: ``dict``
+        """
+        LOG.info('Loading group to role map definitions from "%s"' % (self._role_maps_path))
+        file_paths = self._get_group_to_role_maps_file_paths()
+
+        result = {}
+        for file_path in file_paths:
+            LOG.debug('Loading group to role mapping from: %s' % (file_path))
+            group_to_role_map_api = self.load_group_to_role_map_assignment_from_file(
```

Comment:
Wdym? It follows exactly the same pattern as other loaders.

And failing to load a mapping is fatal anyway so the exception should be propagated - aka fail loudly and not silently ignore real issues.